.env.example.prod

```
@@ -12,11 +12,10 @@ POSTGRES_USER=sjxHBcWufSbTJRVSCynqDavkxHefYxJP
 POSTGRES_PASSWORD=wgnuhcBthzKLm6e9NCUgMZXVlwjvEWNAzTqqA3LAmN7LcXzWmOWxjEgogCQghVMo
 
 AWS_STORAGE_BUCKET_NAME=prr-tracker
-AWS_ACCESS_KEY_ID=minio
-AWS_SECRET_ACCESS_KEY=minio123
+AWS_ACCESS_KEY_ID=terriblkeyid
+AWS_SECRET_ACCESS_KEY=terriblekey
 AWS_S3_REGION_NAME=us-west-2
-AWS_S3_ENDPOINT_URL=http://minio:9000
-MINIO_ROOT_PASSWORD=minio123
+AWS_S3_ENDPOINT_URL=sfo3.digitaloceanspaces.com
```

It turns out (learned this the hard way :sweat_smile:) This has to start with the scheme
```suggestion
AWS_S3_ENDPOINT_URL=https://sfo3.digitaloceanspaces.com
```